### PR TITLE
VNF Scalability Post GA.

### PR DESCRIPTION
**Pull request by BShrivastav**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/vpc/data_source_ibm_is_instance_profiles.go [hunk 1]
```diff
@@ -464,6 +464,31 @@ func DataSourceIBMISInstanceProfiles() *schema.Resource {
...
 						},
+						"network_interface_count": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"max": {
+										Type:        schema.TypeInt,
+										Computed:    true,
+										Optional:    true,
```
Comment (deepaksibm) [inline ibm/service/vpc/data_source_ibm_is_instance_profiles.go:475]: remove optional

File: ibm/service/vpc/data_source_ibm_is_instance_profile.go [hunk 2]
```diff
@@ -461,6 +461,31 @@ func DataSourceIBMISInstanceProfile() *schema.Resource {
...
 			},
+			"network_interface_count": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"max": {
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Optional:    true,
```
Comment (deepaksibm) [inline ibm/service/vpc/data_source_ibm_is_instance_profile.go:472]: here as well remove optional in all 3 places

File: website/docs/d/is_instance_profiles.html.markdown [hunk 3]
```diff
@@ -125,6 +125,15 @@ You can access the following attribute references after your data source is crea
...
     - `value` - (String) The value for this profile field.
     - `values` - (String) The permitted values for this profile field.
+
+  - `network_interface_count` - (List)
+  
+      Nested scheme for `network_interface_count`:
+      - `max` - (Integer) The maximum value for this profile field.
+      - `min` - (Integer) The minimum value for this profile field.
+      - `type` - (String) The type for this profile field.
+        - Constraints: Allowable values are: `range`.
```
Comment (deepaksibm) [inline website/docs/d/is_instance_profiles.html.markdown:135]: remove constraints

File: website/docs/d/is_instance_profile.html.markdown [hunk 4]
```diff
@@ -127,6 +127,13 @@ In addition to the argument reference list, you can access the following attribu
   - `type` - (String) The type for this profile field.
   - `value` - (String) The value for this profile field.
   - `values` - (String) The permitted values for this profile field.
+- `network_interface_count` - (List)
+
+  Nested scheme for `network_interface_count`:
+  - `max` - (Integer) The maximum value for this profile field.
+  - `min` - (Integer) The minimum value for this profile field.
+  - `type` - (String) The type for this profile field.
+      - Constraints: Allowable values are: `range`.
```
Comment (deepaksibm) [inline website/docs/d/is_instance_profile.html.markdown:136]: remove constraints

Comment (BShrivastav) [reply website/docs/d/is_instance_profile.html.markdown]: done

File: website/docs/d/is_instance_profiles.html.markdown (hunk 3 above)
Comment (BShrivastav) [reply website/docs/d/is_instance_profiles.html.markdown]: done

File: ibm/service/vpc/data_source_ibm_is_instance_profile.go (hunk 2 above)
Comment (BShrivastav) [reply ibm/service/vpc/data_source_ibm_is_instance_profile.go]: done

File: ibm/service/vpc/data_source_ibm_is_instance_profiles.go (hunk 1 above)
Comment (BShrivastav) [reply ibm/service/vpc/data_source_ibm_is_instance_profiles.go]: done

File: website/docs/d/is_instance_profile.html.markdown [hunk 5]
```diff
@@ -127,6 +127,12 @@ In addition to the argument reference list, you can access the following attribu
   - `type` - (String) The type for this profile field.
   - `value` - (String) The value for this profile field.
   - `values` - (String) The permitted values for this profile field.
+- `network_interface_count` - (List)
+
+  Nested scheme for `network_interface_count`:
+  - `max` - (Integer) The maximum value for this profile field.
```
Comment (deepaksibm) [inline website/docs/d/is_instance_profile.html.markdown:133]: Should we make it like Maximum network interfaces allowed for this profile ?

**Comment by BShrivastav** [comment]
Closing as Upstream is raised: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4527

**Pull request closed without merging**